web/index.html

```diff
@@ -38,6 +38,23 @@ <h1 class="dib pa3 ma0 lh-tight">
         </form>
     </section>
     <section class="mw8 center lh-copy dark-gray pv4 ph2 ph4-ns">
+        <h2 class="f4">Where do I find my multiaddr?</h2>
+        <ul>
+            <li class="pb2">
+                <strong>Using IPFS Desktop or IPFS WebUI</strong>
+                <ul>
+                    <li>Open the IPFS WebUI "Status" page via the IPFS Desktop menu or by visiting "http://127.0.0.1:5001/webui" (when using the default config settings)

```

Comment:
Renders fine for me but is missing trailing </li>

Reply:
Thanks for the catch 🙏